test/e2e/16.0/wildflyserver_test.go

```diff
@@ -43,3 +36,7 @@ func wildFlyBasicTest(t *testing.T) {
 func wildFlyClusterTest(t *testing.T) {
 	wildflyframework.WildFlyClusterTest(t, "16.0")
 }
+
+func wildFlySmokeRecoveryScaledownTest(t *testing.T) {
```

Comment:
you can remove this test (I kept the WildFly 16 tests just in case) but the WildFly Operator really started working with WildFLy 17 (and its new S2I image)